public/js/helpers/get_startTime.js

```diff
@@ -0,0 +1,25 @@
+/* global axios Cal addDateEventListeners */
+
+(function () {
+  axios
+    .get('https://nazareth-open-tourism-platform.herokuapp.com/api/v1/events')
+    .then(function (res) {
+      var data = res.data.reduce((acc, event) => {
+        return acc.concat(event.startTime);
+      }, []);
```

Comment:
Why is this not a map?

```js
var data = res.data.map(event => event.startTime);
```

Reply:
Also what is this weird mix of ES6 and ES5? I'm guessing with webpack everything will move to lovely ES6 anyway :+1:

Reply:
why should it be a map when reduce can do everything ? :stuck_out_tongue_winking_eye:

Reply:
While I agree that reduce is the best function, map is close behind and more appropriate here :dancer: